go/src/koding/klientctl/main.go

```diff
@@ -424,7 +420,12 @@ func run(args []string) {
 							},
 							cli.StringFlag{
 								Name:  "team, t",
-								Usage: "Specify a koding.com team to log in. Leaving empty logs in to kd.io by default.",
+								Usage: "Specify a Koding team to log in. Leaving empty logs in to kd.io by default.",
+							},
+							cli.StringFlag{
+								Name:  "koding",
```

Comment:
what about baseURL? koding is too general for specifying koding url. there are other tools that uses baseURL too